ParkPlanner/Facilities.cs

```diff
@@ -85,7 +85,7 @@ public partial class Recdatum
 
         [JsonProperty("FacilityID", NullValueHandling = NullValueHandling.Ignore)]
         [JsonConverter(typeof(ParseStringConverter))]
-        public long? FacilityId { get; set; }
```

Comment:
2. Use string for Facility IDs:
    Consider using string instead of long? for better compatibility, if FacilityId, OrgFacilityID, ParentRecAreaID, and other ID-related properties are meant to represent identifiers.

Reply:
Since this is QuickType generated, the source likely doesn't have quotes around this. I'd leave as is.